Source/Blazorise/Components/Validation/Validations.razor.cs

```diff
@@ -127,6 +128,11 @@ internal void NotifyValidationStatusChanged( IValidation validation )
         // Try to come up with solution that StatusChanged will be called only once while it will
         // still provide all of the failed messages.
 
+        if ( Model is IValidatableObject validatableObject )
+        {
+            var result = validatableObject.Validate( new ValidationContext( Model ) );
```

Comment:
We don't need to do anything with the result here? use discard instead then?

Reply:
It was an unnecessary call, so I removed it.